playground/legacy/main.js

```diff
@@ -3,9 +3,9 @@ import viteSvgPath from './vite.svg'
 import MyWorker from './worker?worker'
 
 async function run() {
-  await import('./custom0.js')
-  await import('./custom1.js')
-  await import('./custom2.js')
+  console.log(await import('./custom0.js'))
+  console.log(await import('./custom1.js'))
+  console.log(await import('./custom2.js'))
```

Comment:
These imports actually have no side-effects so Rollup _should have_ been tree-shaking them. However, the way the preload method was previously injected prevented them from getting tree-shaken.

Now that it's being applied correctly, Rollup accurately detects that their export values weren't being used, which is why I had to add `console.log`s here.